app-ng/tests/test-job.lisp

```diff
@@ -0,0 +1,11 @@
+(in-package :qvm-app-ng-tests)
+
+(deftest test-jobs ()
```

Comment:
Maybe add some simple concurrency tests to verify the locking. Nothing too crazy, just spawn a few threads that attempt to create/start/kill/check status on a small pool of jobs concurrently. One thing that I've found helpful when writing concurrency tests is to write the tests & make sure they pass, then disable locking at various points in the code and re-run the tests and make sure they fail!  When I was working on the concurrency tests for the persistent-qvm locking, I was surprised to find that my concurrent access patterns were often insufficient to really stress the locking and detect bugs.